.github/workflows/ci.yml

```diff
@@ -0,0 +1,28 @@
+name: CI
```

Comment:
Example run with artifacts (and a build failure for  a typo. oops!): https://github.com/astarche/grpc-device/actions/runs/1889272848
Example successful PR run: https://github.com/astarche/grpc-device/actions/runs/1866276664